packages/ui/src/components/molecules/DropDownListItem.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { FC, ReactElement, useState } from 'react';
+import styled from 'styled-components';
+
+import { theme } from '../../themes/theme.styled';
+import {
+  CheckBox,
+  Image,
+  LangDisplay,
+  RadioButton,
+  Tag,
+  Typography,
+  TypographyColor,
+  TypographyProps,
+} from '../atoms';
+import { BorderProps, border } from '../utils';
+
+export interface DropDownListItemProps extends BorderProps {
+  leftImageSrc?: string;
+  rightIconSrc?: string;
+  rightText?: string;
+  tag?: string;
+  text: string;
+  displayRadioButton?: boolean;
+  radioButtonValue?: string;
+  restrictedItem?: boolean;
+  rightTextColor?: TypographyColor;
+  shortForm?: string;
+  rightTextVariant?: TypographyProps['variant'];
+  showCheckBox?: boolean;
+  id?: string;
+  onClick?: () => void;
+  selectedItem?: string | undefined;
+  changeColorWhite?: boolean;
+  checked?: boolean;
+  onCheckedChange?: (checked: boolean) => void;
+}
+
+export interface DropDownListItemHorizontalBoxProps {
+  isChecked: boolean;
+  borderRadius?: string; // Change the type to string or use a custom type alias
```

Comment:
This should not be present here, it's already defined in `BorderProps`

Reply:
Removed and changed it